Tests/DatadogTests/Datadog/TracerTests.swift

```diff
@@ -335,8 +335,8 @@ class TracerTests: XCTestCase {
         Datadog.setUserInfo(id: "abc-123", name: "Foo")
         tracer.startSpan(operationName: "span with user `id` and `name`").finish()
 
-        Datadog.setUserInfo(id: "abc-123", name: "Foo", email: "foo@example.com")
-        tracer.startSpan(operationName: "span with user `id`, `name` and `email`").finish()
+        Datadog.setUserInfo(id: "abc-123", name: "Foo", email: "foo@example.com", extraInfo: ["extra_key": "extra_value"])
```

Comment:
Because `extraInfo` encoding depends on the feature (Logging and RUM supports multiple `Encodable` types, but Tracing encodes all values as `String`), it seems better to mix few types in this test:
```swift
extraInfo: ["extra_string": "string_value", "extra_int": 42, "extra_bool": true]
```
and document the expected encoding result in test assertions.